src/main/scala/Day4/PrimeChecker.scala

```diff
@@ -0,0 +1,28 @@
+//package Day4
+//
+//object PrimeChecker extends App {
```

Comment:
are you continuing with this or should it be deleted?

Reply:
continuing